tests/plugins/plugin_loader_tests.py

```diff
@@ -73,6 +73,7 @@ def test_load_generators_finds_all_generators(plugin_loader):
 
 def test_load_generators_finds_correct_evaluators(plugin_loader):
     evaluators = plugin_loader.load_evaluators()
-    assert set(evaluators.keys()) == {'spawn_evaluator', 'multiply_evaluator'}
+    assert 'spawn_evaluator' in evaluators
+    assert 'multiply_evaluator' in evaluators
```

Comment:
The plugin loaded will interpret everything imported from `spawn/plugins/json_input_file.py` as an evaluator. Seems a bit loose way of doing things - I don't know if we can improve this in the future

Reply:
I think that was on purpose to facilitate the development of evaulators - no fancy code etc

Reply:
true - review at a later date I suppose